### ISSUE TITLE
Product Popularity Item Descriptions cut off.

### ISSUE THREAD

**Issue by DevDreed** [Team-Slytherin/Bangazon-Terminal-Interface#89]
### Feature Description
The column for item description is being cut off.

**Comment by tayloraharry** [comment]
the requirements in the readme required a certain width on the columns, but if it doesn't matter, we can make as wide as we want

**Comment by DevDreed** [comment]
Yeah we can cancel this issue.  I just let my wife play around with the app to see what she found and she noticed that.  I didn't realize that it was in the requirements.
